src/core/server/index.ts

```diff
@@ -74,6 +74,7 @@ export {
   IsAuthenticated,
   KibanaRequest,
   KibanaRequestRoute,
+  LifecycleResponseFactory,
```

Comment:
Shouldn't we prefix these types with `Http...` to stick with our convention to avoid naming conflicts and make them more discoverable? (Also applies to the other HTTP types, so would probably make sense to do it in a different PR)

Reply:
babel v7.5+ [supports typescript namespaces](https://github.com/babel/babel/pull/9785). probably we can make another attempt to switch to them. for now, we can use manual convention.

Reply:
Well that sure is exciting.